pkgs/development/python-modules/tidalapi/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, python-dateutil
+, requests
+, ffmpeg-python
+, keyring
+, pillow
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "tidalapi";
+  version = "0.7.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-LdlTBkCOb7tXiupsNJ5lbk38syKXeADvi2IdGpW/dk8=";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    python-dateutil
+  ];
+
+  checkInputs = [
+    ffmpeg-python
+    keyring
+    pillow
+    pytest
+  ];
+
+  meta = with lib; {
```

```suggestion
  meta = with lib; {
    changelog = "https://github.com/tamland/python-tidal/releases/tag/v${version}";
```

We require the changelog to be referenced, if one exists.